lux/action/univariate.py

```diff
@@ -55,6 +55,7 @@ def univariate(ldf, *args):
         recommendation = {
             "action": "Distribution",
             "description": "Show univariate histograms of <p class='highlight-descriptor'>quantitative</p>  attributes.",
+            "long_description": "Distribution displays univariate histogram distributions of all quantitative attributes (e.g., Horsepower, Displacement). Visualizations are ranked from most to least skewed.",
```

Comment:
`Horsepower, Displacement` is still not adaptive.

Reply:
Whoops, fixed, my commits didn't get push for some reason.